challenge_2/python/alexbotello/main.py

```diff
@@ -0,0 +1,6 @@
+if __name__ == '__main__':
+    N = int(input()) # How many items in the list?
+    array = [int(input()) for _ in range(N)] # Enter ints
+
+    print("\nThe single number is ", end='')
+    print(*[i for i in array if array.count(i) == 1], sep=' or ')
```

Comment:
Probably not; the method will still probably look through every element. What if you counted all the elements up front? Could you iterate over the counts and fine the element that showed up once?